src/js/components/Form/stories/FieldStatesCustom.js

```diff
@@ -0,0 +1,138 @@
+import React, { useRef, useEffect } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, Form, FormField, TextInput, Grommet } from 'grommet';
+import { deepMerge } from '../../../utils';
+
+const customTheme = deepMerge(grommet, {
+  formField: {
+    border: {
+      error: {
```

Comment:
This seems like it should be `error.border`, to align with `error.background`.

Reply:
We'll track this one on https://github.com/grommet/grommet/issues/4004

Reply:
👍